hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/config/ConfigServer.java

```diff
@@ -25,48 +28,63 @@
 import java.util.TreeSet;
 
 public class ConfigServer {
+  private final Log log = LogFactory.getLog(ConfigServer.class);
 
   private Server server;
   private Engine engine;
-  private SchedulerConf schedulerConf;
```

Comment:
We should be consistent and name this hdfsFrameworkConfig, just as it is named elsewhere.